core/trino-main/src/test/java/io/trino/type/TestDecimalOperators.java

```diff
@@ -194,14 +214,17 @@ public void testDivide()
         assertDecimalFunction("DECIMAL '100.00' / DECIMAL '0.3'", decimal("0333.33"));
         assertDecimalFunction("DECIMAL '100.00' / DECIMAL '0.30'", decimal("00333.33"));
         assertDecimalFunction("DECIMAL '100.00' / DECIMAL '-0.30'", decimal("-00333.33"));
+        assertDecimalFunction("DECIMAL '-100.00' / DECIMAL '0.30'", decimal("-00333.33"));
```

Comment:
how this is testing negative 0 case?

Reply:
These are additional test cases I added for completeness. I can move them to a separate commit.